doc/data-mining-library/source/reference/code/imputation-default.py

```diff
@@ -0,0 +1,7 @@
+from Orange.data import Table
+from Orange.preprocess import Impute
+
+data = Table("heart-disease.tab")
+imputer = Impute()
+
+impute_heart = imputer(data)
```

Comment:
missing newline at EOF
(a properly configured text editor should take care of these automatically)